### PR TITLE
(Issue 1209) Use pathname instead of path to get youtube video id

### PR DESCRIPTION
**Pull request by mrvisser**
Fix for #1209

### CODE REVIEW

File: node_modules/oae-preview-processor/tests/test-previews.js [hunk 1]
```diff
@@ -1003,7 +1003,7 @@ describe('Preview processor', function() {
                         _verifySignedUriDownload(restCtx, content.previews.mediumUrl, function() {
 
                             // Assert that short youtube links are processed as well
-                            _createContentAndWait('link', 'http://youtu.be/lgTQ5I_H4Xk', null, function(restCtx, content) {
+                            _createContentAndWait('link', 'http://youtu.be/lgTQ5I_H4Xk?t=130', null, function(restCtx, content) {
```
Comment (nicolaasmatthijs) [inline node_modules/oae-preview-processor/tests/test-previews.js]: Should we have a test for both the "no querystring" and "querystring" scenario?

Comment (mrvisser) [inline node_modules/oae-preview-processor/tests/test-previews.js]: I had considered it but didn't feel like it was worth it. Not too passionate either way but I just opted to inflate test time as little as possible.

**Comment by mrvisser** [comment]
I added a second test. Back to @simong

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged.

**Pull request merged**